// NOTE: remember to add to config.h:
/* +	{ MODKEY,             XK_f,      toggletile,  {0} }, */
--- dwm/dwm.c
+++ dwm/dwm.c
@@ -211,6 +211,7 @@ static void tagmon(const Arg *arg);
 static void togglebar(const Arg *arg);
 static void togglefloating(const Arg *arg);
 static void togglesticky(const Arg *arg);
+static void togglefullscr(const Arg *arg);
 static void toggletag(const Arg *arg);
 static void toggleview(const Arg *arg);
 static void unfocus(Client *c, int setfocus);
@@ -2029,6 +2051,13 @@ togglesticky(const Arg *arg)
 	arrange(selmon);
 }
 
+void
+toggletile(const Arg *arg)
+{
+	if (!selmon->sel)
+		return;
+	if (selmon->sel->isfullscreen) /* no support for fullscreen windows */
+		return;
+	selmon->sel->isfixed = !selmon->sel->isfixed || selmon->sel->isfloating;
+	if (selmon->sel->isfixed)
+		/* restore last known float dimensions */
+		resize(selmon->sel, selmon->sel->sfx, selmon->sel->sfy,
+		       selmon->sel->sfw, selmon->sel->sfh, False);
+	else {
+		/* save last known float dimensions */
+		selmon->sel->sfx = selmon->sel->x;
+		selmon->sel->sfy = selmon->sel->y;
+		selmon->sel->sfw = selmon->sel->w;
+		selmon->sel->sfh = selmon->sel->h;
+	}
+	arrange(selmon);
+}
+
 void
 toggletag(const Arg *arg)
 {
